cron/syncAlertsToMongo.py

```diff
@@ -6,47 +6,25 @@
 # Copyright (c) 2014 Mozilla Corporation
 
 import calendar
-import logging
 import random
 import sys
 from datetime import datetime
 from configlib import getConfig, OptionParser
-from logging.handlers import SysLogHandler
 from pymongo import MongoClient
 
+from mozdef_util.utilities.logger import logger, initLogger
 from mozdef_util.utilities.toUTC import toUTC
 from mozdef_util.elasticsearch_client import ElasticsearchClient
 from mozdef_util.query_models import SearchQuery, TermMatch
 
-logger = logging.getLogger(sys.argv[0])
-
-
-def loggerTimeStamp(self, record, datefmt=None):
-    return toUTC(datetime.now()).isoformat()
-
-
-def initLogger():
-    logger.level = logging.INFO
-    formatter = logging.Formatter(
-        '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-    formatter.formatTime = loggerTimeStamp
-    if options.output == 'syslog':
-        logger.addHandler(
-            SysLogHandler(
-                address=(options.sysloghostname, options.syslogport)))
-    else:
-        sh = logging.StreamHandler(sys.stderr)
-        sh.setFormatter(formatter)
-        logger.addHandler(sh)
-
 
 def genMeteorID():
     return('%024x' % random.randrange(16**24))
 
 
 def getESAlerts(es):
     search_query = SearchQuery(minutes=50)
-    search_query.add_must(TermMatch('_type', 'alert'))
+    search_query.add_must(TermMatch('type', 'alert'))
```

Comment:
In https://github.com/mozilla/MozDef/pull/1129 we aren't setting the type here to alert anymore, so this logic will then be broken.